macos/FluentUI/Button/Button.swift

```diff
@@ -121,6 +122,37 @@ open class Button: NSButton {
 		}
 	}
 
+	func setupBorderShadowsIfNeeded() {
+		// At the moment border shadows only apply for the Primary and Secondary Button Styles.
+		// Border shadows being meant for buttons and their corresponding States, that have ~some~
+		// background color other than `clear` which is why they don't apply to the Borderless Style
+		// buttons. Acrylic Style buttons do possess a background color, yet they maintain transparency.
+		// This transparency allows any underlying border shadow layers to bleed through, consequently
+		// impacting the overall button background color. Adding shadows in this context presents a
```

Comment:
I think this makes total sense, thanks for writing! And if I'm not mistaken, the acrylic style is meant to mimic some system buttons that have a visual effect and no shadow so makes sense.